core/src/test/java/org/apache/accumulo/core/conf/PropertyTest.java

```diff
@@ -250,7 +250,7 @@ public void validatePropertyKeys() {
 
   @Test
   public void testAnnotations() {
-    assertTrue(Property.GENERAL_VOLUME_CHOOSER.isExperimental());
+    assertFalse(Property.GENERAL_VOLUME_CHOOSER.isExperimental());
```

Comment:
This change removes the only test case for the isExperimental() method returning true. Instead of negating the assertion, this should have changed the property to one that still returns true.

Reply:
Fixed in 86c842ea919fa7fd68314783d530878e3510b085